core/utils.py

```diff
@@ -133,9 +133,31 @@ def open_file(
     return file
 
 
+@overload
+def get_file_contents(filepath: str) -> str: ...
+
+
+@overload
+def get_file_contents(
+    filepath: str, *, mode: str = 'r'
+) -> str: ...
+
+
+@overload
+def get_file_contents(
+    filepath: str, *, raw_bytes: Literal[False], mode: str = 'r'
+) -> str: ...
+
+
+@overload
+def get_file_contents(
+    filepath: str, *, raw_bytes: Literal[True], mode: str = 'r'
+) -> bytes: ...
+
+
 def get_file_contents(
-        filepath: str, raw_bytes: bool = False, mode: str = 'r'
-) -> bytes:
+    filepath: str, raw_bytes: bool = False, mode: str = 'r'
+) -> Union[str, bytes]:
```

Comment:
Why this change? Can it be standardized to bytes only? If not then maybe you'll need some explanation in the return docstring of when it'll be str and when it'll be bytes.

Reply:
Docstring is changed PTAL!

Reply:
> Why this change? Can it be standardized to bytes only? 

There are places in the codebase where string values are expected from this method but since it is annotated with bytes only, sometimes a conflict is showed up. So, to cover those conflicts we did this change.

A small discussion regarding this can be found here https://github.com/oppia/oppia/pull/15762#discussion_r924619654

Reply:
OK, yup -- with the overrides, this looks fine. Thanks!